assets/src/application/ui/layout/entityList/EntityList.tsx

```diff
@@ -56,7 +56,10 @@ const EntityList = <E extends Entity, ELFS extends EntityListFilterStateManager<
 				listId={listId}
 			/>
 			{entityList}
-			<EntityPagination filterState={filterState} />
+			{
+				// disable pogination when sorting
+				!filterState.sortingEnabled && <EntityPagination filterState={filterState} />
```

Comment:
interesting and when sorting is done `sortingEnabled` is set to false?

Reply:
Yeah, when user disables it or switches to card view